components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -357,6 +361,20 @@ function mapToGRPCError(err: any): any {
     return err;
 }
 
+function getUserAgentAndClientName(call: grpc.ServerUnaryCall<any,any>) {
+    const clientNameHeader = call.metadata.get("client-name")
+    const userAgentHeader = call.metadata.get("user-agent")
+
+    let [clientName, userAgent] = ["", ""]
+    if (clientNameHeader.length != 0) {
+        clientName = clientNameHeader[0].toString()
+    }
+    if (userAgentHeader.length != 0) {
+        userAgent = userAgentHeader[0].toString()
+    }
+    return {clientName, userAgent}
```

Comment:
is it possible to also log ip address here? for example, I assume if someone is using old gpctl, it will not send `client-name`, and maybe not even `user-agent`. it would be good to at least log some sort of breadcrumb.

Reply:
I initially refrained from adding peer information because it was returning `127.0.0.1`. However, I have added it now.